### ISSUE TITLE
Please remove the THREE.js requirement for including altspace.js

### ISSUE THREAD

**Issue by JoshGalvin** [AltspaceVR/AltspaceSDK#83]
Currently you have to include three.js even if you don't have 3D elements.

```html
    <script src="http://sdk.altvr.com/libs/three.js/r71/build/three.min.js"></script>
    <script src="http://sdk.altvr.com/libs/altspace.js/latest/altspace.min.js"></script>
```

Ideally you could just include altspace and make use of the firebase connection for interesting 2D applications in VR.

**Comment by brianpeiris** [comment]
Good point. We should be able to make Three.js optional for 2D apps. I've added this to our backlog.

**Comment by GavanWilhite** [comment]
That may be tricky given that a number of the utilities use three.js classes, but worth thinking about. Perhaps making it easier to use the individual utilities would be a way to go.
